package.json

```diff
@@ -66,10 +66,33 @@
     "semver": "^5.6.0"
   },
   "peerDependencies": {
-    "eslint": "^3.0.0 || ^4.0.0 || ^5.0.0"
+    "eslint": "^4.0.0 || ^5.0.0"
   },
   "engines": {
-    "node": ">=8.x",
+    "node": ">=10.x",
```

Comment:
Why make this breaking change? Babel works in node 6.

Reply:
It's because I changed the babel-preset-env config to target node 10. What are your thoughts on what node versions we should support? I've just been going off of what node stable currently is.

Reply:
I would support always as far back as possible. As a Babel plugin, I’d say you have an obligation to support every node version Babel does, for as long as Babel supports it.